asserts/checkers.go

```diff
@@ -92,3 +93,27 @@ func checkUint(headers map[string]string, name string, bitSize int) (uint64, err
 	}
 	return value, nil
 }
+
+func checkCommaSepList(headers map[string]string, name string) ([]string, error) {
+	listStr, ok := headers[name]
+	if !ok {
+		return nil, fmt.Errorf("%q header is mandatory", name)
+	}
```

Comment:
It may not be worth it, but would it make sense to split checkMandatory into two, e.g. checkMandatory & checkEmpty, so that checkMandatory only has one responsibility and can be reused here?